src/services/proxies/bulk-resolver.service.ts

```diff
@@ -7,6 +7,21 @@ export class BulkProxy<C extends any, R extends ResolverService> implements Reso
 
   private readonly configuredResolvers: R[] = [];
 
+  private readonly notFoundMessages = [
```

Comment:
`allowableErrorMessages` or sth like this
They are not all for `not found`

Reply:
fixed